api/mesh/v1alpha1/mesh.proto

```diff
@@ -20,6 +20,7 @@ message Mesh {
   option (kuma.mesh.resource).kds.send_to_zone = true;
   option (kuma.mesh.resource).ws.name = "mesh";
   option (kuma.mesh.resource).ws.plural = "meshes";
+  option (kuma.mesh.resource).short_name = "m";
```

Comment:
Can this break on-wire compatibility? Like if a global is using X and user is using X-1?

Reply:
Seems like a similar PR previously did not change anything special or add UPGRADE.md notes so I think it's fine https://github.com/kumahq/kuma/pull/12109/files#diff-ebf17957528e5b0c92852cadc86a5c16e9ebfd594731b109e52f82c7cce59811

Reply:
yeah I think we never use these `(kuma.mesh.resource)` in KDS right? It's probably used only when we're generating ResourceTypeDescriptor. So should be safe without UPGRADE.md